tests/unit/pubsub_v1/subscriber/test_streaming_pull_manager.py

```diff
@@ -550,6 +550,26 @@ def make_running_manager(**kwargs):
     )
 
 
+def await_manager_shutdown(timeout=None):  # pragma: NO COVER
+    # NOTE: This method should be called after manager.close(), i.e. after the shutdown
+    # thread has been started.
+    shutdown_thread = next(
```

Comment:
I would rather we store the shutdown thread in a private variable in streaming_pull_manager and read it that way. It seems a bit cleaner than enumerating through all the threads, and is probably more performant, too.